### DISCUSSION TITLE
Copilot using a old account!<title>

### DISCUSSION THREAD

**Discussion by JossiePrincipesa2k22** [community/community#40328] [Copilot Conversations]
### Select Topic Area

Show & Tell

### Body

If you are installing Github/Copilot  in Neovim; and it keeps promting aut with a old account:
   --Go to Neovim command promt and  type:"Copilot logout"
  -- Type at the prompt again "Copilot login'

That works!

**Comment by ossanna16** [comment]
Hi there @JossiePrincipesa2k22 and welcome to our community! Thank you for sharing that great workaround with us 🙂

**Reply by AmadeusSystemm** [reply]
Invalid param logout

**Reply by ebsi-bblake** [reply]
Copilot signout (you can also do Copilot help)
